projects/lib/src/configurator/editors/condition-editor.component.html

```diff
@@ -8,29 +8,29 @@
 
   <div class="mb-2">
     <label class="form-label" for="data" i18n>Data</label>
-    <select [(ngModel)]="context.config.condition.data" id="data" class="form-select" (ngModelChange)="updateConfig()">
+    <select [(ngModel)]="context.config?.condition.data" id="data" class="form-select" (ngModelChange)="updateConfig()">
```

Comment:
There is a `*ngIf="context.config.condition"` above which means that if `context.config` is undefined there will be an error anyway.